test/e2e/framework/managed/kubeapi.go

```diff
@@ -30,7 +25,7 @@ import (
 
 // KubernetesApiFixture manages a kubernetes api server
 type KubernetesApiFixture struct {
-	EtcdUrl   *url.URL
+	Etcd      *integration.Etcd
```

Comment:
(No action required) Why have you decided to switch to running an etcd instance per api server (vs the old strategy of reusing a single etcd instance)?  

I'm inclined to give this issue a pass anyway (as per #718), but I was curious as to why you decided to make this change.

Reply:
When i upgraded the kubebuilder to v1.0.8, the binaries which came along with it kube-apiserver (v1.13.1) no longer works with `--secure-port=0`. So instead of updating and fixing the arguments, i thought it is best to use the methods from the imported package. Also since it is just the test fixture, so i believe it does not hurt spawning multiple instances of etcd.

Reply:
It's memory overhead and would limit the maximum number of clusters we could test with.  But, not a long-term concern.